### DISCUSSION TITLE
Subscribing to comments and posts

### DISCUSSION THREAD

**Discussion by Jeb166** [Docile-Alligator/Infinity-For-Reddit#1001] [Duplicate Ideas] [answered]
It's a really useful feature that exists in the official Reddit app. For example i could subscribe to a comment and get notifications if someone replies to it, same for the posts. Is it possible to add this feature into the Infinity?

**Comment by Wladefant** [comment, marked as answer]
Duplicate of #963. Consider adding your idea there
